test/hotspot/jtreg/vmTestbase/nsk/share/MainWrapper.java

```diff
@@ -46,7 +46,8 @@ public static void main(String[] args) throws Throwable {
         Finalizer finalizer = new Finalizer(new FinalizableObject());
         finalizer.activate();
 
-
+        // Some tests use this property to understand if virtual threads are used
+        System.setProperty("main.wrapper", "Virtual");
```

Comment:
Shouldn't this be:

`System.setProperty("main.wrapper", wrapperName);`

Reply:
Thanks, better to use wrapperName as a well as to set it it in TestScaffold.